guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -0,0 +1,33 @@
+[id="remediating-compliance-failures_{context}"]
+= Remediating compliance failures
+
+You can examine compliance reports and, in some cases, remediate cases of non-compliance.
+
+[WARNING]
+====
+Always test the recommended remedial actions or scripts in a non-production environment before implementing them in production.
+Remediation might render the system non-functional.
+====
+
+.Prerequisites
+* Your user account has a role assigned that has the `view_arf_reports` and `view_hosts` permissions.
+
+.Procedure
+To examine a simplified compliance policy report:
+
+. In the {ProjectWebUI}, navigate to *Hosts* > *Compliance* > *Reports*.
+. In the *Reported At* column, click the time link of the report you want to examine.
+{Project} displays a simplified list of policy rules with the results of the scan.
+
+To examine a full compliance policy report and remediate compliance failures manually:
```

Comment:
If you say "manually", is there an automated way too? If there is, please add a link.